source/JsMaterialX/JsMaterialXCore/JsDocument.cpp

```diff
@@ -59,8 +60,17 @@ EMSCRIPTEN_BINDINGS(document)
         .function("getTypeDefs", &mx::Document::getTypeDefs)
         .function("removeTypeDef", &mx::Document::removeTypeDef)
         BIND_MEMBER_FUNC("addNodeDef", mx::Document, addNodeDef, 0, 3, stRef, stRef, stRef)
-        BIND_MEMBER_FUNC("addNodeDefFromGraph", mx::Document, addNodeDefFromGraph, 7, 7, const mx::NodeGraphPtr, 
-            stRef, stRef, stRef, bool, stRef, std::string)
+        BIND_MEMBER_FUNC("addNodeDefFromGraph", mx::Document, addNodeDefFromGraph, 4,4, const mx::NodeGraphPtr, const std::string,
```

Comment:
For now, I think it's fine to omit deprecated wrappers in JavaScript, since we don't yet have a base of MaterialX JavaScript clients.  Once we begin seeing significant usage of the JavaScript bindings, though, this looks like a good pattern that we can consider adopting.

Reply:
Done